CMakeSettings.json

```diff
@@ -2,97 +2,97 @@
       "installRoot": "${env.USERPROFILE}\\CMakeBuilds\\${workspaceHash}\\install\\${name}",
       "cmakeCommandArgs": "",
-      "buildCommandArgs": "-m -v:minimal",
+      "buildCommandArgs": "",
       "variables": [
         {
-          "name": "OPENSSL_OFF",
+          "name": "INSTALLER_MERGE_MODULES",
           "value": "OFF"
         },
         {
-          "name": "CMAKE_BUILD_TYPE",
+          "name": "ENABLE_SQL",
+          "value": "OFF"
+        },
+        {
+          "name": "CMAKE_BUILD_TYPE_INIT",
           "value": "RelWithDebInfo"
         },
         {
-          "name": "USE_SYSTEM_OPENSSL",
-          "value": "OFF"
+          "name": "WIN32",
+          "value": "WIN32"
         },
         {
-          "name": "PORTABLE",
-          "value": "ON"
+          "name": "ENABLE_LIBRDKAFKA",
+          "value": "OFF"
         },
         {
-          "name": "USE_SYSTEM_ZLIB",
+          "name": "ENABLE_JNI",
           "value": "OFF"
         },
         {
-          "name": "DISABLE_CONTROLLER",
-          "value": "TRUE"
+          "name": "OPENSSL_OFF",
+          "value": "OFF"
         },
         {
-          "name": "USE_SYSTEM_UUID",
+          "name": "ENABLE_COAP",
           "value": "OFF"
         },
         {
           "name": "USE_SHARED_LIBS",
           "value": "OFF"
         },
         {
-          "name": "FORCE_WINDOWS",
+          "name": "DISABLE_CONTROLLER",
           "value": "ON"
         },
         {
-          "name": "DISABLE_ROCKSDB",
-          "value": "FALSE"
-        },
-        {
-          "name": "FAIL_ON_WARNINGS",
-          "value": "OFF"
-        },
-        {
-          "name": "ENABLE_WEL",
-          "value": "TRUE"
+          "name": "BUILD_ROCKSDB",
+          "value": "ON"
         },
         {
-          "name": "ENABLE_SQL",
-          "value": "TRUE"
+          "name": "FORCE_WINDOWS",
+          "value": "ON"
         },
         {
-          "name": "ENABLE_LIBRDKAFKA",
+          "name": "USE_SYSTEM_UUID",
           "value": "OFF"
         },
         {
-          "name": "DISABLE_CURL",
+          "name": "DISABLE_LIBARCHIVE",
           "value": "OFF"
         },
         {
-          "name": "FORCE_WINDOWS",
+          "name": "DISABLE_SCRIPTING",
           "value": "ON"
         },
         {
-          "name": "ENABLE_COAP",
-          "value": "OFF"
+          "name": "EXCLUDE_BOOST",
+          "value": "ON"
         },
         {
-          "name": "DISABLE_LIBARCHIVE",
+          "name": "ENABLE_WEL",
           "value": "TRUE"
         },
         {
-          "name": "DISABLE_SCRIPTING",
-          "value": "TRUE"
+          "name": "FAIL_ON_WARNINGS",
+          "value": "OFF"
         },
         {
-          "name": "EXCLUDE_BOOST",
-          "value": "TRUE"
+          "name": "SKIP_TESTS",
+          "value": "OFF"
+        },
+        {
+          "name": "STRICT_GSL_CHECKS",
+          "value": "AUDIT"
```

Comment:
I suggest `ON` by default instead of `AUDIT`. Audit checks shouldn't be enabled anywhere other than the CI by default, IMO.

https://github.com/gsl-lite/gsl-lite/#contract-checking-configuration-macros

Reply:
done